shared/common-adapters/badge.native.js

```diff
@@ -6,9 +6,10 @@ import {globalStyles, globalColors} from '../styles'
 
 import type {Props} from './badge'
 
-function Badge ({badgeStyle, badgeNumber, badgeNumberStyle}: Props) {
+function Badge ({badgeStyle, badgeNumber, badgeNumberStyle, outlineColor}: Props) {
+  const outlineStyle = outlineColor ? {borderWidth: 3, borderColor: outlineColor} : {border: 'none'}
```

Comment:
React native does not have a "border" property. Does an empty object `{}` suffice here? Or `borderWidth: 0`?